### ISSUE TITLE
Accessibility Issue Role and selected state are not announced for color controls

### ISSUE THREAD

**Issue by shashankpatel1542** [pnp/sp-dev-fx-controls-react#1943]
User Experience:
Screen reader dependent users will be impacted as the Role and selected state are not announced for color controls. Due to this, users will not be able to identify the purpose of the control. This will confuse them.

Actual Result:
Role and selected state are not announced for color controls.

Like: at "Theme Darker blue" color control->
**NVDA** announces as: "table Theme darker row 1 column 1".
**Narrator** announces as: "No item in view".

Observation:
Narrator is not announcing the 'name' of color controls.
Expected Result:
Proper name, role and 'selected' state should be announced for color controls.
Like: Screen reader can announce: "Theme colors grouping, Theme darker blue button, selected 1 of 10"

See attached screenshot for your reference for Font Color:

![Image](https://github.com/user-attachments/assets/5adcedbd-c522-46a9-9209-85c97c576337)

As you can see NVDA is not announcing state for the selected color which will confuse the screen reader users.

**See attached screenshot for your reference for Highlighted Color:**

![Image](https://github.com/user-attachments/assets/1db0af2f-5b2b-48cc-bc1f-3e50ecc293b5)

**As you can see Aqua color is selected still its announcing as not selected.**
